supply/lib/supply/uploader.rb

```diff
@@ -25,11 +25,21 @@ def perform_upload
 
       upload_binaries unless Supply.config[:skip_upload_apk]
 
+      promote_track if Supply.config[:track_promote_to]
+
       UI.message("Uploading all changes to Google Play...")
       client.commit_current_edit!
       UI.success("Successfully finished the upload to Google Play")
     end
 
+    def promote_track
```

Comment:
The Play API does not allow upload of an APK version code if it already exists as active version on another track.    

The use case we have (and I think it may be typical) is to upload APK to a test track (we use beta), test it, and finally promote that version to production.  This cannot be done with existing tool (as far as I can tell).  Even if it were possible to upload same version to another track - it is convenient to NOT have to upload the same APK a second time.  In our situation (with 60+ apps) that can take much longer than just commanding the API to promote the APK between tracks.

The current supply framework really centers on the act of "performing an upload".  Commanding the Play API to promote already uploaded assets between tracks doesn't fit perfectly.  I tried to fit it best I could while solving the real problem I had.  Consider it a functional feature that is open to improvement to anyone with ideas.

Summarized feature:
"--track_promote_to=" works with the "--track=" parameter to command the Play API to promote existing Play track APK version(s) (--track param value) to a new track (--track_promote_to value).